traffic_ops/testing/api/v4/servers_test.go

```diff
@@ -950,6 +954,11 @@ func UpdateTestServers(t *testing.T) {
 		t.Fatalf("Cannot test server type change update; server '%s' had nil type ID", hostName)
 	}
 
+	// Check to verify mtu changed
+	if originalMTU == *respServer.Interfaces[0].MTU {
+		t.Errorf("MTU value didn't update. Expected: #{*updatedMTU}, actual: #{*originalMTU}")
```

Comment:
I don't think this is going to print out the values, I think this will just literally print out
```
MTU value didn't update. Expected: #{*updatedMTU}, actual: #{*originalMTU}
```

Reply:
ok